project2/styles.css

```
@@ -0,0 +1,194 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300|Roboto:300,400,500,900&display=swap');
+
+* {
+    box-sizing: border-box;
+}
+
+.u-button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+.clearfix::before,
+.clearfix::after {
+    content: ' ';
+    display: table;
+}
+
+.clearfix::after {
+    clear: both;
+}
+
+body {
+    color: rgb(66, 30, 35);
+    font-family: 'Roboto Condensed', sans-serif;
+    background-color: rgb(88, 52, 56);
+}
+
+h2 {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-size: 19.5px;
+    line-height: 16.px;
```

The extra period here, without being followed by a number, causes this rule to break/not be used:
```suggestion
    line-height: 16px;
```